tests/test_features/test_histogram.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from tests.base import VisualTestCase
+from tests.dataset import DatasetMixin
+from yellowbrick.features.histogram import *
+
+##########################################################################
+## BalancedBinningReference Tests
+##########################################################################
+
+class TestBalancedBinningReference(VisualTestCase, DatasetMixin):
+	"""
+	Test the BalancedBinningReference visualizer 
+	"""
```

Comment:
Thank you for adding this doc string to make the tests more readable (the test output depends on the docstring)!